tests/TemporalExpression/DayInWeekTest.php

```diff
@@ -29,7 +29,7 @@ public function constructor_UsingInvalidDayValue_ShouldThrowAnException($day)
     /**
      * @test
      */
-    public function includesDate_WhenProvidedDateAtSameMonthDay_ShouldReturnTrue()
+    public function includesDate_WhenProvidedDateAtSameWeekDay_ShouldReturnTrue()
```

Comment:
still not the good name, your SUT is includes not includesDate, WhenProvided => Given